components/combobox/combobox.jsx

```diff
@@ -102,6 +102,10 @@ const propTypes = {
 		PropTypes.object,
 		PropTypes.string,
 	]),
+	/**
+	 * TODO - Remove this unused prop in the next major release
+	 */
+	direction: DIRECTIONS.LTR,
```

Comment:
Go ahead and move this from the propTypes, and we can make it undocumented, but still be in the code.